frontend/frontend/src/components/SavedArtist.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+	<div class="artists">
+		<div class="artist-bin">
+			<div class="left">
+				<p>{{ artistName }}</p>
+			</div>
+			<div class="right">
+				<button
+					class="saveButton"
+					v-on:click="
+						savedArtist
+							? updateArtistRelationship('remove')
+							: updateArtistRelationship('save')
+					"
+				>
+					{{ savedArtist ? "UNFAVOURITE" : "FAVOURITE" }}
+				</button>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import axios from "axios";
+export default {
+	props: ["artistName"],
+	data() {
+		return {
+			savedArtist: false,
```

Comment:
I think you should keep a local array in the vuex store called artists or something and when the person logs in we can grab the names of artists they have favourited, then update it locally when we update the database. this way we can set the savedArtist variable based on if the vuex store variable .includes() the artistName